### PR TITLE
Update map commands to handle new GeoLite2 block geolocation format

### PR DESCRIPTION
**Pull request by srkirkland**
Updated to using new MaxMind "GeoLite2-City-CSV" format.  This requires a new (free) access token and the lat/lng are now stored on the blocks and not on the location file.  This makes the location file and resulting table not needed anymore.

Updated `setup_database.sh` to handle the new download and extraction of the GeoLite2 files, including conversion to an int based CIDR range that the app used.

Updated `generate_ip_database.py` to only read the blocks file and store that in SQLite

Updated `plot.py` to read lat/lng info from the new location in the blocks table.

### CODE REVIEW

**Review by gassmoeller — COMMENTED** [review]
Looks reasonable to me. Have you tested if it works now?

File: map/ip_database/generate_ip_database.py [hunk 1]
```diff
@@ -107,14 +75,12 @@ def main():
 
     if wipe_db == "y":
         DB_NAME = sys.argv[1]
-        LOCATION_CSV_FILE = sys.argv[2]
+        LOCATION_CSV_FILE = sys.argv[2] #unused
```
Comment (gassmoeller) [inline map/ip_database/generate_ip_database.py:78]: Feel free to remove if not used anymore (we can always recover it later if necessary).

**Comment by srkirkland** [comment]
> Looks reasonable to me. Have you tested if it works now?

I have tested that I can run it on the `geoweb.cse.ucdavis.edu` server in a new folder and that it'll populate the ip_lookup_db with good ip range and lat/lng values.  But the /home/backend directory is read-only and I don't see any cron jobs setup on that server either (the only one I have access to), so that's all the testing that I know how to do.

**Comment by gassmoeller** [comment]
Ah, yes the cron job is run on `shell.geodynamics.org`. It has the same directory /home/backend/cig_backend mounted. Let me know if you need access to that.

**Comment by srkirkland** [comment]
> Ah, yes the cron job is run on `shell.geodynamics.org`. It has the same directory /home/backend/cig_backend mounted. Let me know if you need access to that.

Yes please -- I'll run the test in a different directory and on a copy of the code but at least on `shell` i'll have the correct python versions and modules and crontab so I can make sure the environment is the same.

**Comment by gassmoeller** [comment]
Ok, lets merge this and see if it fixes things. I will report back what I see on shell.geodynamics.org.

**Pull request merged**